tests/dataimportcron_test.go

```diff
@@ -154,6 +156,7 @@ var _ = Describe("DataImportCron", Serial, func() {
 				},
 			}
 			snapshot = utils.WaitSnapshotReady(f.CrClient, snapshot)
+			Expect(snapshot.Labels).To(HaveKeyWithValue(testKubevirtIoKey, testKubevirtIoValue))
```

Comment:
Again not sure how this fails. Maybe it's possible to reach DV succeeded before the labels are copied from prime? does this consistently pass locally?

Reply:
Added 2fa5f271be170bfb0f5f795006061b558b132faf to hopefully mitigate this.